injector/src/main/java/edu/ucr/cs/riple/injector/Helper.java

```diff
@@ -149,10 +148,14 @@ private static TypeDeclaration<?> findTopLevelClassDeclarationOnCompilationUnit(
       return annotationDeclaration.get();
     }
     Optional<ClassOrInterfaceDeclaration> interfaceDeclaration = tree.getInterfaceByName(name);
-    return interfaceDeclaration.orElseGet(() -> tree.getClassByName(name).orElse(null));
+    if (interfaceDeclaration.isPresent()) {
+      return interfaceDeclaration.get();
+    }
+    throw new TargetClassNotFound("Top-Level", name, tree);
   }
 
-  public static Node findTopLevelDirectInnerClass(Node cursor, String name) {
+  private static Node findTopLevelDirectInnerClass(Node cursor, String name)
```

Comment:
Even though these are private methods, I think they would benefit from Javadocs. What is the difference between a "Top level direct inner class" and a "top level inner class", for example?

Reply:
Added javadocs for all methods in this class. I am planning to add javadoc through every PR :) c623aacf3634525f48470ead8e8d3506348029d6

Direct inner class requires the class to be connected to parent directly, and the other one does not have this constraint. `top level` name is also removed from them, they are on `findDirectInnerClass` and `findInnerClass`.